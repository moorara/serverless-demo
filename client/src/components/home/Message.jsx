import React from 'react'

const Message = ({ message }) => (
  <section className="hero is-info is-medium">
    <div className="hero-body">
      <div className="container">
        <h1 className="title">Message Panel</h1>
        <div className="content">
          <p>{message}</p>
        </div>
      </div>
    </div>
  </section>
)

export default Message
