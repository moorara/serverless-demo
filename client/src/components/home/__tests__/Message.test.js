import React from 'react'
import renderer from 'react-test-renderer'

import Message from '../Message'

describe('Message', () => {
  test('renders react component', () => {
    const component = renderer.create(<Message />)
    const tree = component.toJSON()
    const instance = component.root

    expect(tree).toMatchSnapshot()
  })
})
