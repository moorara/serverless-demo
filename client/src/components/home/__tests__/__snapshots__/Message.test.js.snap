// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message renders react component 1`] = `
<section
  className="hero is-info is-medium"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        Message Panel
      </h1>
      <div
        className="content"
      >
        <p />
      </div>
    </div>
  </div>
</section>
`;
